src/Event/EventSubscriber.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
The namespace for event subscribers is inside an `EventSubscriber` folder. Also, the name of this class needs to be a bit more descriptive. It can subscribe to multiple events (and we will actually do that), but simply `EventSubscriber` is not enough. Let's call it `EcasEventSusbcriber` or something . 

We should also check that somehow we are only doing this in case email is missing or we are using ECAS (since this is not an issue for now with other servers). We could maybe do this by setting an attribute on the property bag when we subscribe to the other event (post validate that we created a patch for). This way we know we need to set an email.